pyro/contrib/epidemiology/compartmental.py

```diff
@@ -390,9 +391,13 @@ def fit_svi(self, *,
             model_trace = poutine.trace(
                 poutine.replay(particle_plate(model), guide_trace)).get_trace()
             self.samples = {name: site["value"] for name, site in model_trace.nodes.items()
-                            if site["type"] == "sample"}
+                            if site["type"] == "sample"
```

Comment:
Interesting, I didn't know we could have multiple `if`s in dict/list comprehensions. 🙂

Reply:
Yeah, it's a nice way to break up a long `and` condition.